axis/vendor.styl

```diff
@@ -50,14 +50,16 @@ border-radius()
 //
 // ex. opacity: .6
 
-opacity(opacity, args...)
-  
-  if args  
-    filter: s("progid:DXImageTransform.Microsoft.Alpha(Opacity=%s)", opacity * 100 args) if support-for-ie  
+opacity($opacity, $args...)
+
+  $support-for-ie = support-for-ie // address in #192
```

Comment:
The only other place I had to do this. Hoping we can just drop all the vendor stuff soon anyway...